envs/env.py

```diff
@@ -404,18 +412,39 @@ def reset(self) -> GymObs:
         initial_state = np.array([self.state_statistics[State(param.value)][Statistic(stat.value)]
                                   for param in State for stat in Statistic])
         self.acked_bytes = 0
+        self.current_mice_flows_kbs = self.bandwidth_start * .08
+        self.current_traffic_patterns = [self.bandwidth_start * 125  * .4, self.bandwidth_start * 125  * .8, self.bandwidth_start * 125  * .4, self.bandwidth_start * 125  * .208]
```

Comment:
Values in index 0 and 2 are the same, is it intentional?

Reply:
Yes, it's the same flow, but we should move all the traffic patterns logic such that is given by the network generator